scripts/sct_deepseg.py

```diff
@@ -37,7 +40,15 @@ def get_parser():
     input_output = parser.add_argument_group("\nINPUT/OUTPUT")
     input_output.add_argument(
         "-i",
-        help="Image to segment.",
+        nargs="+",
+        help="Image to segment. Can be multiple images (separated with space).",
+        metavar=Metavar.file)
+    input_output.add_argument(
+        "-c",
+        nargs="+",
+        help="Type of image contrast. Indicates the order in which the images have been presented with -i. "
```

Comment:
i understand the need for this argument to be mandatory in case of 2+ input files, but why making it mandatory for single input? As of now, the contrast information is already included in `-task`, e.g. `-task seg_sc_t2star`.

Also, in the future, we might have models that accommodate several contrasts, in which case inputting the contrast will be irrelevant.

Reply:
It is not mandatory if there is only one input (see [here](https://github.com/neuropoly/spinalcordtoolbox/pull/3041/files#diff-3cb3eafa0ea1f4fbd635239caf0ecf415e67062f0e35a638b72cecb8cb1dc892R152)). I can explicit this in the help of this flag.

Currently, if there are more than 1 contrast the flag `-c` is required, so for now it doesn't accommodate the fact that we might have a HeMIS model (optional multichannel model). It is not too late to choose option 1 (no flag `-c`) and trust that the user will read the models' description.

Reply:
OK that makes sense. I got confused by why you would [modify our current models](https://github.com/neuropoly/spinalcordtoolbox/blob/25f60b28f032e98872c27880e8ba64b9b900507e/spinalcordtoolbox/deepseg/models.py#L20) which only require a single input, but you're probably doing it to accommodate potential future cascading of these existing models. So i agree with these proposed changes 👍 

And yes: mentioning that `-c` is only required for tasks with 2+ models would be useful.

Reply:
> OK that makes sense. I got confused by why you would [modify our current models](https://github.com/neuropoly/spinalcordtoolbox/blob/25f60b28f032e98872c27880e8ba64b9b900507e/spinalcordtoolbox/deepseg/models.py#L20) which only require a single input, but you're probably doing it to accommodate potential future cascading of these existing models.

Yes exactly and also to display the input contrast(s) needed with `-list-tasks`.


> And yes: mentioning that `-c` is only required for tasks with 2+ models would be useful.

I added the info here: 25f60b2